REPL.cpp

```diff
@@ -140,34 +170,150 @@ bool REPL::ExecuteSwift(std::string line)
     if(ShouldLog(LoggingPriority::Info))
     {
         Log("=========AST Before Modifications==========");
-        src_file->dump();
+        tmp_src_file->dump();
     }
-    AddImportNodes(*src_file, m_modules);
-    //TODO(sasha): Load DLLs
-    swift::performNameBinding(*src_file);
+    AddImportNodes(*tmp_src_file, m_modules);
+
+    swift::performNameBinding(*tmp_src_file);
     CHECK_ERROR();
     swift::TopLevelContext top_level_context;
     swift::OptionSet<swift::TypeCheckingFlags> type_check_opts;
-    swift::performTypeChecking(*src_file, top_level_context, type_check_opts);
+    swift::performTypeChecking(*tmp_src_file, top_level_context, type_check_opts);
     
-    ModifyAST(*src_file);
+    ModifyAST(*tmp_src_file);
     
-    //TODO(sasha): Perform playground transform?
```

Comment:
Was this TODO taken care of? Looks like you might have been unsure of it.

Reply:
There's a function called `performPlaygroundTransform` which adds some calls to external logging functions - I'd added that TODO to wait to decide if we needed to do that. I don't think we do, but I may just be unenlightened.